load_vbr_box.m

```diff
@@ -0,0 +1,57 @@
+function vbr_predictions = load_vbr_box(box_file, qmethod, observations)
+    %%%
+    % Parameters
+    % ----------
+    %
+    % box_file 
+    %   file path to box to load, e.g., 'sweep_box.mat'
```

Comment:
ya -- one option is to check if `box_file` exists, load it if it does and if it doesn't we can call the `generate_sweep_` function and save the result so that it can be loaded directly next time. Either that, or if `box_file` doesn't exist we can display an error message with instructions for how to generate the parameter sweep.

Reply:
add a to do